.github/workflows/code_scan.yaml

```diff
@@ -109,12 +109,13 @@ jobs:
         uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
         with:
           python-version: "3.10"
+      - name: Install uv
+        run: |
+          pip install uv
       - name: Install tox
         run: |
-          python -m pip install --require-hashes --no-deps -r .ci/requirements.txt
-          pip-compile --generate-hashes --output-file=/tmp/requirements.txt --extra=ci_tox pyproject.toml
-          python -m pip install --require-hashes --no-deps -r /tmp/requirements.txt
-          rm /tmp/requirements.txt
+          uv pip install -r .ci/requirements.txt
```

Comment:
What's the advantage of using `uv pip` instead of `uv sync`?

Reply:
It also creates a virtual environment automatically

Reply:
I'm nore sure as we don't have uv.lock. Do you think we should use uv sync?

Reply:
Let's deal with it later